src/search/views.py

```diff
@@ -3,6 +3,7 @@
 from django.db.models import Q
 from django.shortcuts import redirect, render
 from ctirs.models import Group, Feed, STIPUser as User
+from feeds.views import extraction_tags, create_link_tags
```

Comment:
No `extraction_tags` in feeds.view.py.
Replace to `extract_tags`.

Reply:
I corrected it with https://github.com/s-tip/stip-sns/pull/95/commits/71bd1739eaf213bf427d4e28fb251efbfa1da365.